sql/sqldb-or-3rdparty.md

```diff
@@ -150,28 +150,50 @@ By design and default, with no configuration required to activate, Azure SQLDB r
 
   Business Continuity Overview
   https://azure.microsoft.com/en-us/documentation/articles/sql-database-business-continuity
+
+Another point to consider is the retention period of saved backups: in Azure SQLDB, at the moment, automatic backups created by the infrastructure have a maximum retention period of 35 days for Point-in-Time-Restore (PITR). If you need a longer period, you should use SQL Server installed in Azure Virtual Machines.
+
+
+**HYBRID: You need to integrate with on-premise AlwaysOn Availability Group (AG) topology?**
```

Comment:
If you wouldn't mind changing  "on-premise" to "on-premises" that would be great. It's in at least two places. Just want to make sure we get that right. :)